solutions/usaco-1233.mdx

```diff
@@ -0,0 +1,76 @@
+---
+id: usaco-1233
+source: USACO Gold 2022 Open
+title: Apple Catching
+author: Qi Wang
+---
+
+[Official Analysis (Java)](http://www.usaco.org/current/data/sol_prob1_gold_open22.html)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N \log N)$
+
+<LanguageSection>
+
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+
+using namespace std;
+
+int N, ans = 0;
+map<int, int> mpp;
+struct item {
+    int q, t, x, n;
+} p[200005];
+
+bool cmp(const item& x, const item& y) {
+    if (x.x - x.t == y.x - y.t)
+        return x.q > y.q;
+
+    return x.x - x.t < y.x - y.t;
+}
+
+int main() {
+	cin >> N;
+	for(int i = 0; i < N; i++){
+        cin >> p[i].q >> p[i].t >> p[i].x >> p[i].n;
+	}
```

Comment:
sus

Reply:
oops

Reply:
hold up thats sus it looks fine on my editor

Reply:
does vs code not have convert spaces to tabs anymore???

Reply:
<img width="477" alt="image" src="https://user-images.githubusercontent.com/55369003/208350566-e363c9c3-0fad-4e67-b7ab-a77764106ccf.png">
idk man, seems fine on my computer